CMakeLists.txt

```diff
@@ -944,6 +948,9 @@ if(NOT SKIP_LIBRARY)
 		add_library(${BINARY_NAME} STATIC ${SRC})
 	endif()
 
+	include(GenerateExportHeader)
+	generate_export_header(${BINARY_NAME} BASE_NAME mgba STATIC_DEFINE BUILD_STATIC EXPORT_FILE_NAME include/mgba-util/dllexports.h)
```

Comment:
Apologies for missing this the first time around and thus giving you the wrong advice, but this should be exported to `${CMAKE_CURRENT_BINARY_DIR}/include/mgba-util`, which may need to be created manually.